openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -1115,9 +1115,8 @@
 
   #team_concurrency_notice {
     margin: ($baseline-v/2) 0;
-    border: 1px solid $color-error;
-    background-color: $color-error-back;
-    color: $color-error;
+    border: 1px solid #ffd875;
```

Comment:
Match these colors to their edX theme counterparts and define as variables in `openassessment/xblock/static/sass/themes/edx/_utilities.scss`

Reply:
fair.  I noticed there were other colors defined in this file, but can also add to theme.  thought that was a shared theme

Reply:
are we confident those labels don't override others?

Reply:
The named variables only live w/in ORA.